README.md

```diff
@@ -1,49 +1,52 @@
 # Introduction
 
-The Exchange Union Daemon ([`xud`](https://github.com/ExchangeUnion/xud)) is the open-source node software powering [Exchange Union](https://www.exchangeunion.com/), a decentralized exchange (DEX) built on the [Lightning](https://lightning.network/) and [Raiden](https://raiden.network/) network.
+The Exchange Union Daemon ([`xud`](https://github.com/ExchangeUnion/xud)) is the reference node implementation powering [OpenDEX](https://opendex.network), a decentralized exchange (DEX) built on top of the [Lightning](https://lightning.network/) and [Raiden](https://raiden.network/) network. `xud` brings individual traders, market makers and exchanges onto OpenDEX to form a single global trading network.
 
-`xud` brings individuals and exchanges onto the same network to form a single global liquidity pool. This gives users a choice to either trade directly on the DEX by running `xud`, managing private keys and software stack, *or* to conveniently trade via a trusted exchange. Exchanges benefit from access to the network's aggregated liquidity and can provide deeper order books and new trading pairs to their users.
+**Traders** use [OpenDEX-enabled exchanges](https://opendex.network/trade/exchanges) as gateways to trade on OpenDEX. These exchanges are usually non-custodial and abstract away from the technological complexity of the DEX and offer a superior UX by enabling trading via a simple website. **Exchanges** use `xud` to hedge user trades on OpenDEX in order to lock in profits. Liquidity on OpenDEX is provided by **market makers**, who use `xud` to take orders from external exchanges (e.g. Binance) and offer these on OpenDEX for a premium. Exchange Union provides SDK's for both, exchanges to automate hedging with `xud` and for market makers to connect to external exchanges and automate the arbitrage trades.
```

Comment:
Second to last sentence would read better (and flow with the previous two sentences) as "**Market Makers** use `xud` to arbitrage with external exchanges (e.g. Binance) and provide liquidity on OpenDEX for a premium."

Reply:
Thanks! Replaced the clunky sentence with your version.